README.md

```diff
@@ -1 +1,46 @@
-# Webpack
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-do list project.
+
+> This is the javaScript project for Microverse. In this project, I built a simple to-do list useing HTML, CSS, javaScript and WebPack.
+
+## Built With 
+
+- HTML
+- JavaScript
+- CSS
+- WebPack
+
+## Getting Started
+
+- Open up GitHub.
+- Clone this repository.
+- Check out any file you are interested in.
```

Comment:
[Required] please add the following sections to your readme file, it's important that you provide information on how other developers or future collaborators can clone and run this project on their computers. 
kindly add detailed instructions to the following sections of your readme file with instructions on cloning your repository and setting it up locally.
- Prerequisites
- Setup
- Install